tests/wifi_test.go

```diff
@@ -19,11 +17,11 @@ func TestAllClustersAppWiFi(t *testing.T) {
 
 	t.Cleanup(func() {
 		utils.SnapRemove(t, allClustersSnap)
-		utils.SnapDumpLogs(nil, start, allClustersSnap)
+		utils.SnapDumpLogs(t, start, allClustersSnap)
 	})
 
 	// Install all clusters app
-	utils.SnapInstallFromStore(t, allClustersSnap, utils.ServiceChannel)
+	utils.SnapInstallFromStore(t, allClustersSnap, "latest/edge")
```

Comment:
What is the reason for hard coding `edge` here and not use what is defined by `ServiceChannel`? Maybe a note in the comment will help us remember.

Reply:
It should have never been `utils.ServiceChannel`. That is to choose the snap under testing, i.e. Chip Tool, and not the testing components.

Reply:
That makes sense, but the followup question would be: Why don't you use `stable`? I know the answer is matter sdk version compatibility, but maybe worth noting it in the comment.

Reply:
The default value is "latest/edge", regardless of the current change. Same as what is set in Thread tests.
Sure, it should be changed to something with a lower risk level: https://github.com/canonical/chip-tool-snap/issues/71